openpype/hosts/houdini/plugins/publish/collect_cache_farm.py

```diff
@@ -0,0 +1,24 @@
+# if farm data is off it will skip
+#  if farm data is true it will submit to farm publish
+import pyblish.api
+
+
+class CollectDataforCache(pyblish.api.InstancePlugin):
+    """Collect data for caching to Deadline."""
+
+    order = pyblish.api.CollectorOrder
+    families = ["ass", "pointcache",
+                "vdbcache", "mantraifd",
+                "redshiftproxy"]
+    hosts = ["houdini"]
+    targets = ["local", "remote"]
+    label = "Collect Data for Cache"
+
+    def process(self, instance):
+        if instance.data.get("farm"):
+            instance.data["families"].append("publish.hou")
+            data = dict()
+            data["chunkSize"] = 100
```

Comment:
This seems arbitrary? Surely there are better ways to make sure that the exports that run in one go actually produce a single task in deadline?

I recall seeing a "Submit as single task" or something setting in Deadline but couldn't quickly find it in the current arguments. But that's what I'd at least expect to create a workflow where we force it a single task in a clear distinct way (separate from ChunkSize I'd say unless we consider setting chunkSize to e.g. 999999 safe enough).

Reply:
I will make this into chuckSize: 9999999 just to be safe